Loop/AppDelegate.swift

```diff
@@ -92,7 +93,9 @@ final class AppDelegate: UIResponder, UIApplicationDelegate, DeviceOrientationCo
     }
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-        
+
+        FirebaseApp.configure()
```

Comment:
This should be behind a feature flag for DIY users who are not using this (probably the majority), and it will be turned off by default for DIY.

Reply:
Ok, will add one.